src/surveys.rs

```diff
@@ -0,0 +1,82 @@
+use std::collections::{HashMap, BTreeMap};
+
+use serde::{Serialize, Deserialize};
+
+use crate::UserGroups;
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq)]
+pub enum PossibleAnswers {
+    Input(u16),
+    MultipleChoice(HashMap<u16, (String, String)>),
+    OneChoice(HashMap<u16, (String, String)>),
+    Priority(HashMap<u16, (String, String)>),
+    Value { min: f64, max: f64, step: f64 },
+}
+
+impl Default for PossibleAnswers {
+    fn default() -> Self {
+        Self::Input(12)
+    }
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
+pub enum Answer {
+    Input(String),
+    MultipleChoice(Vec<u16>),
+    OneChoice(u16),
+    Priority(Vec<(u16, u16)>),
+    Value(f64),
+}
+
+impl Default for Answer {
+    fn default() -> Self {
+        Self::Input(String::new())
+    }
+}
+    
+
+#[derive(Serialize, Clone, Debug, PartialEq)]
+pub struct Survey {
+    pub id: String,
+    pub question: (String, String),
+    pub answers: PossibleAnswers,
+    pub date_start: i64,
+    pub date_end: i64,
```

Comment:
Ce sont des timestamp pas des dates. Il faudrait qu'on prennent l'habitude de les noter avec le suffixe ts

```suggestion
    pub start_ts: i64,
    pub end_ts: i64,
```